src/cdk/drag-drop/drag.ts

```diff
@@ -491,6 +491,19 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
       // Preserve the previous `transform` value, if there was one.
       this._rootElement.style.transform = this._initialTransform ?
           this._initialTransform + ' ' + transform : transform;
+
+      // Apply transform as attribute if dragging and svg element to work for IE
+      if (this._rootElement instanceof SVGElement) {
+        let appliedTransform = getComputedStyle(this._rootElement).getPropertyValue('transform');
```

Comment:
The intent here was to only preserve the inline style `transform` since we replace the entire `style.transform` and getting the attribute value is fairly cheap. Doing a `getComputedStyle` introduces a potential performance issue.

Reply:
Ok, so I changed it to use the `activeTransform` values.  Is that appropriate?